src/test/java/com/epam/tc/hw2/ex2/GeneralWebDriver.java

```diff
@@ -0,0 +1,8 @@
+package com.epam.tc.hw2.ex2;
+
+import org.openqa.selenium.WebDriver;
+
+public class GeneralWebDriver {
```

Comment:
what is a goal of making this class?

Reply:
This class was created to make WebDriver common for all test classes. Accordingly, each of the classes works with the current page, which is stored in this variable.